tests/Feature/LibUnitTests/PhotosUnitTest.php

```diff
@@ -46,6 +46,7 @@ public function upload(
 		$response = $this->testCase->post(
 			'/api/Photo::add', [
 				'albumID' => $albumID,
+				'fileLastModifiedTime' => 1678824303000,
```

Comment:
what happens when this value is 0 ?
Another test to add is to also check if we can have an image without that creation date exif data and check that the associated value is as expected.

Reply:
Sure, I'll add one. The expected behavior would be that in case of 0, the create date will be 1st January, 1970 (because we're processing the fileLastModifiedTime as an UNIX timestamp).